db/seeds.rb

```diff
@@ -135,1049 +135,6 @@
     spitcast_id: 205,
     spitcast_slug: 'malibu-malibu-ca',
     name: 'Malibu' },
-  { subregion: LA,
```

Comment:
Hmm, why'd you remove all of these seeds?

Reply:
My mistake, it was for debugging purposes. Fixed!